packages/governance/src/votes/votes.cairo

```diff
@@ -165,7 +159,7 @@ pub mod ERC20VotesComponent {
             let is_valid_signature_felt = ISRC6Dispatcher { contract_address: delegator }
                 .is_valid_signature(hash, signature);
 
-            // Check either 'VALID' or true for backwards compatibility.
+            // Check either 'VALID' or True for backwards compatibility.
```

Comment:
We've been using True in some places, but we should refactor them into true, since that's the correct way of writing the value in cairo.

Reply:
Makes sense. At some point during the PR it got changed